packages/app-mobile/components/NoteEditor/EditorType.ts

```diff
@@ -0,0 +1,36 @@
+// Types related to the NoteEditor
```

Comment:
Only class names should be PascalCase. Type files we often just call them "types.ts". Or if you need to be more specific, "editorTypes.ts" (plural for consistency)